modules/ocf_kubernetes/manifests/master/loadbalancer.pp

```diff
@@ -33,8 +33,17 @@
   } ~>
   service { 'keepalived': }
 
+  $vip = 'lb-kubernetes'
+  $cnames = ldap_attr($vip, 'dnsCname', true)
+  $alias_redirects = flatten($cnames.map |$domain| {
+    prefix(['.ocf.io', ''], $domain).map |$fqdn| {
```

Comment:
i would add a comment about why the empty prefix is there, as it is not immediately obvious. A comment in general about what this is doing would be good.

Reply:
declare `vip` here and move the following block to be directly above the `haproxy::frontend` where it is actually used

Reply:
I wanted to move this to be closer, but I was worried that it would break the arrow relationshp between `ocf_kubernetes::master::loadbalancer::ssl` and the `haproxy::frontend`. Not sure how this can be done within Puppet syntax.

Reply:
It is done with `require`